hermes/probe/monitor.go

```diff
@@ -95,7 +95,7 @@ func (mp *MonitorProbe) Init(name string, opts *options.Options) error {
 //	- ctx: context provided for cancelling probe.
 //	- metricChan: unidirectional channel used for sending metrics to be surfaced.
 func (mp *MonitorProbe) Start(ctx context.Context, metricChan chan<- *cpmetrics.EventMetrics) {
-	probeTicker := time.NewTicker(mp.opts.Interval)
+	probeTicker := time.NewTicker(time.Duration(mp.config.GetIntervalSec()) * time.Second)
```

Comment:
I would add a method to MonitorProbe that returned the execution interval as a time.Duration value.

Reply:
Ack.